etc/schema/mysql.schema.sql

```diff
@@ -87,6 +87,7 @@ CREATE TABLE x509_target (
   latest_certificate_chain_id int(10) unsigned NULL DEFAULT NULL,
   ctime timestamp NULL DEFAULT NULL,
   mtime timestamp NULL DEFAULT NULL ON UPDATE CURRENT_TIMESTAMP,
+  last_scan bigint unsigned NOT NULL,
```

Comment:
`ctime` and `mtime` should be the last columns in a table. Please move `last_scan` before them. This also counts for PostgreSQL and the migration script.